echo/mock.go

```diff
@@ -11,7 +11,7 @@ import (
 // mockAuthMiddleware doesn't actually authenticate since we never store auth data.
 // It simply ensures that the requested user exists in the DB and then attaches the
 // users's ID to the context.
-func MockMiddleware(next echo.HandlerFunc) echo.HandlerFunc {
```

Comment:
I think this should remain in auth package, echo pkg shouldn't be interacting with db and the logic should be in `Auth` package along with other auth middlewares

Reply:
This is due to an import cycle